system/inc/system_cloud.h

```diff
@@ -56,9 +56,43 @@ void system_set_time(time_t time, unsigned param, void* reserved);
 
 typedef enum
 {
-	BOOLEAN = 1, INT = 2, STRING = 4, DOUBLE = 9
```

Comment:
It's compatible - we have a suite of API tests that still compile.  The names are now assigned to class constants instead, `INT`, `STRING` etc. so we get more complete type info about what the corresponding variable type should be.